chia/cmds/coins.py

```diff
@@ -64,20 +60,20 @@ def list_cmd(
     fingerprint: int,
     id: int,
     show_unconfirmed: bool,
-    min_coin_amount: str,
-    max_coin_amount: str,
-    excluded_coin_ids: Tuple[str],
-    excluded_coin_amounts: Tuple[int],
+    min_amount: str,
+    max_amount: str,
+    exclude_id: Tuple[str],
+    exclude_amount: Tuple[int],
```

Comment:
Not an error in this PR, but I think these would be lists not tuples.

```suggestion
    excluded_ids: Tuple[str],
    excluded_amounts: Tuple[int],
```